docs/user/installation.rst

```diff
@@ -228,21 +228,21 @@ To check whether it is found by Theano, run the following command:
 It will print ``True`` if everything is fine, or an error message otherwise.
 There are no additional steps required for Theano to make use of cuDNN.
 
-Docker images
-=============
+Docker
+======
 
-Instead of manually installing Theano and Lasagne on your machines as described
-above, you may want to use a pre-made `Docker <https://www.docker.com/whatisdocker>`_
-image: `Lasagne Docker (CPU) <https://hub.docker.com/r/kaixhin/lasagne/>`_ or
+Builds of Lasagne are available as `Docker <https://www.docker.com>`_
+images: `Lasagne Docker (CPU) <https://hub.docker.com/r/kaixhin/lasagne/>`_ or
```

Comment:
OK will do. In terms of the link, it's actually changed from https://www.docker.com/whatisdocker to https://www.docker.com/what-docker, so even though there's a redirect the URL seems like it could keep changing (unlike the top-level website). But I'll put in the new address as they will hopefully keep redirects in place.